operations/helm/charts/mimir-distributed/values.yaml

```
@@ -1712,12 +1723,33 @@ chunks-cache:
   statefulStrategy:
     type: RollingUpdate
 
+  # -- Add extended options for chunks-cache memcached container. The format is the same as for the memcached -o/--extend flag.
+  # Example:
+  # extraExtendedOptions: 'tls,modern,track_sizes'
```

Modern, track sizes is already there.

```suggestion
  # extraExtendedOptions: 'tls'
```

---

i cared about the CSV format. I'll commit this 
```suggestion
  # extraExtendedOptions: 'tls,no_hashexpand'
```